doc/man/scons.xml

```diff
@@ -6732,10 +6778,10 @@ def foo(target, source, env, for_signature):
     return "bar"
 
 # Will expand $BAR to "bar baz"
-env=Environment(FOO=foo, BAR="$FOO baz")
+env = Environment(FOO=foo, BAR="$FOO baz")
 </programlisting>
 
-<para>As a reminder, this evaluation happens when
+<para>As a reminder, substitution evaluation happens when
```

Comment:
I'd change to just substitution and drop evaluation?

Reply:
yeah, sounds good.

Reply:
updated